src/create-label/types/public-response.ts

```diff
@@ -0,0 +1,91 @@
+export type Response = Label;
+
+export type LabelStatus = "processing" | "completed" | "error" | "voided";
+export type Currency = "usd" | "cad" | "aud" | "gbp" | "eur" | "nzd";
+export type LabelChargeEvent =
+  | "carrier_default"
+  | "on_creation"
+  | "on_carrier_acceptance";
+export type LabelFormat = "pdf" | "png" | "zpl";
+export type DisplayScheme = "label" | "qr_code";
+export type LabelLayout = "4x6" | "letter";
+export type TrackingStatus = "unknown" | "in_transit" | "error" | "delivered";
+export type WeightUnit = "pound" | "ounce" | "gram" | "kilogram";
+export type DimensionUnit = "inch" | "centimeter";
+
+export interface Label {
```

Comment:
@j-r-j @pierce-h This might be a bit of a nitpick, but from a user's perspective it's a bit of a discrepancy that we have all of these properties marked as optional even though in the openapi spec and json-schema they are marked as `equired. 

Seems to be an issue with the `json-schema-to-typescript` generator, would it be best to normalize this in the `format-reponse` method?